### PR TITLE
Add proto representation of attestation record for selective revocation

### PR DESCRIPTION
**Pull request by divegeek**

### CODE REVIEW

**Comment by eranmes** [comment]
Looks good to me. @divegeek , when you're back, can you confirm this is the version you want to merge? Then I'll  go ahead and merge.

**Comment by divegeek** [comment]
LGTM.  It appears that I can't merge it myself because I'm not a collaborator.

File: server/src/main/proto/android_attestation.proto [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+
+    repeated KeyPurpose purposes = 1;
+
+    // Specifies the cryptographic algorithm with which the key is used.
+    enum Algorithm {
+      ALGORITHM_UNSPECIFIED = 0;
+      RSA = 1;
+      EC = 3;
+      AES = 32;
+      TRIPLE_DES = 33;
```
Comment (carmenyh) [inline server/src/main/proto/android_attestation.proto:95]: Why 33 for Triple DES?

Comment (divegeek) [reply server/src/main/proto/android_attestation.proto:95]: That's the value from Algorithm.aidl.  Asymmetric algorithms are 1-31, block ciphers 32-127, MAC algorithms 128+.

File: server/src/test/java/com/google/android/attestation/ProtoConverterTest.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
+/* Copyright 2022, The Android Open Source Project, Inc.
```
Comment (carmenyh) [inline server/src/test/java/com/google/android/attestation/ProtoConverterTest.java:1]: How isProtoConverterTest being run? The test BUILD file doesn't change here.

**Pull request closed without merging**